test/libyul/yulOptimizerTests/fullSuite/ssaReverse.yul

```diff
@@ -43,7 +43,6 @@
 //         let a_4, b_4 := abi_decode_bytes_calldata(a_3, b_3)
 //         let a_5, b_5 := abi_decode_bytes_calldata(a_4, b_4)
 //         let a_6, b_6 := abi_decode_bytes_calldata(a_5, b_5)
-//         mstore(a_6, b_6)
```

Comment:
Needs to be modified?

Reply:
Why did you just replace `mstore` by `sstore` in these tests instead of creating new ones?

Reply:
Because the purpose of the test is to optimize the computation. The change from mstore to sstore just prevents the optimizer to remove the computation altogether.